src/controls/default_controls.ts

```diff
@@ -1,117 +1,122 @@
-//@ts-nocheck
-import { FabricObject } from '../shapes/object.class';
+// @ts-nocheck
 
-(function (global) {
-  var fabric = global.fabric,
-    controlsUtils = fabric.controlsUtils,
-    scaleSkewStyleHandler = controlsUtils.scaleSkewCursorStyleHandler,
-    scaleStyleHandler = controlsUtils.scaleCursorStyleHandler,
-    scalingEqually = controlsUtils.scalingEqually,
-    scalingYOrSkewingX = controlsUtils.scalingYOrSkewingX,
-    scalingXOrSkewingY = controlsUtils.scalingXOrSkewingY,
-    scaleOrSkewActionName = controlsUtils.scaleOrSkewActionName,
-    objectControls = FabricObject.prototype.controls;
+import { fabric } from '../../HEADER';
+import { FabricObject } from '../shapes/object.class';
+import {
+  changeWidth,
+  rotationStyleHandler,
+  rotationWithSnapping,
+  scaleCursorStyleHandler,
+  scaleOrSkewActionName,
+  scaleSkewCursorStyleHandler,
+  scalingEqually,
+  scalingXOrSkewingY,
+  scalingYOrSkewingX,
+} from './actions';
+import { Control } from './control.class';
 
-  objectControls.ml = new fabric.Control({
+export const defaultControls = {
+  ml: new Control({
     x: -0.5,
     y: 0,
-    cursorStyleHandler: scaleSkewStyleHandler,
+    cursorStyleHandler: scaleSkewCursorStyleHandler,
```

Comment:
We can't rename those. are still part of the public api. Is not that the name is so bad that needs to be changed.

Reply:
Ahh I messed up exports?

Reply:
https://github.com/fabricjs/fabric.js/pull/8400/files/d435c6b4a0dc1682aa74ee5c97a2710cb0f86e41#r1008768595

Reply:
They were renamed in the file as local vars
I simply undid that

Reply:
ohhh ok i didn't see that.